test/common/math.h

```diff
@@ -1,7 +1,14 @@
 #include <cxxtest/TestSuite.h>
 
 #include "common/math.h"
+/* Macro function for asserting that the compared values x and y
+ * aren't further appart from each other than z.
```

Comment:
Appart -> apart

Also, this multi-line comment formatting is not used in other places

Reply:
Thanks for the comment, I changed the formatting to only use '//'. I hope it is all right now.